File: year2020/month09/day0903/ms_0903.cs
using System;

namespace algorithm_49
{
    class Program
    {
        static void DrawRec(int n) 
        {
            int count = 1;

            for(int i = 0; i < n; i++) 
            {
                for(int j = 0; j < n; j++) 
                {
                    Console.Write(count);
                    count++;

                    if(j != n)
                        Console.Write(' ');
                }

                Console.WriteLine();
            }
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(0 < n && n < 100) 
            {
                DrawRec(n);
            }
        }
    }
}

File: year2020/month09/day0903/ms_0903.cpp
#include <iostream>

using namespace std;

void DrawRec(int n);

int main() {
	int n;

	cin >> n;

	if (0 < n && n < 100) {
		DrawRec(n);
	}

	return 0;
}

void DrawRec(int n) {
	int count = 1;

	for (int i = 0; i < n; i++) {
		for (int j = 0; j < n; j++) {
			cout << count;
			count++;

			if (j != n)
				cout << ' ';
		}

		cout << endl;
	}
}